molgenis-web/src/main/java/org/molgenis/web/PluginInterceptor.java

```diff
@@ -78,6 +78,7 @@ public void postHandle(HttpServletRequest request, HttpServletResponse response,
 			modelAndView.addObject(PluginAttributes.KEY_AUTHENTICATED, SecurityUtils.currentUserIsAuthenticated());
 			modelAndView.addObject(PluginAttributes.KEY_PLUGIN_ID_WITH_QUERY_STRING,
 					getPluginIdWithQueryString(request, pluginId));
+			modelAndView.addObject(PluginAttributes.KEY_PLUGIN_DESCRIPTION, molgenisPlugin.getDescription(pluginId));
```

Comment:
Seeing plugin id and plugin description makes me wonder why there is no plugin label?